docs/dictionary/function/revOpenDatabase.lcdoc

```diff
@@ -50,12 +51,12 @@ Example:
 get revOpenDatabase("postgresql", "192.168.1.100", "dbName", myUsr, myPass, "sslmode=require", "sslcompression=0")
 
 Parameters:
-host (string):
+dbHost (string):
```

Comment:
@seaniepie I think `host:port` should be combined into a single `address` parameter description.